src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -201,6 +203,10 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             // Checks are in the LanguageParser unless otherwise noted.
             switch (feature)
             {
+                // C# 7.3 features.
+                case MessageID.IDS_FeatureAttributesOnBackingFields: // semantic check
+                    return LanguageVersion.CSharp7_2; // TODO Rebase and use LanguageVersion.CSharp7_3 before merging
```

Comment:
:memo: I'll rebase on top of the LanguageVersion.CSharp7_3 change before merging.